doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -242,22 +334,21 @@ To specify particular parameters for ``amp.initialize``, you can use the ``apex_
         }
     )
 
-Note that if using a custom training function, you will need to manage loss scaling manually.
+Note that if using a custom training operator (:ref:`raysgd-custom-training`), you will need to manage loss scaling manually.
 
 
 Distributed Multi-node Training
 -------------------------------
 
-You can scale out your training onto multiple nodes without making any modifications to your training code. To train across a cluster, simply make sure that the Ray cluster is started.
+You can scale out your training onto multiple nodes without making any modifications to your training code. To train across a cluster, simply make sure that the Ray cluster is started. You can start a Ray cluster `via the Ray cluster launcher <autoscaling.html>`_ or `manually <using-ray-on-a-cluster.html>`_.
```

Comment:
```suggestion
You can scale your training to multiple nodes without making any modifications to the training code. To train across a cluster, simply make sure that the Ray cluster is started. You can start a Ray cluster `via the Ray cluster launcher <autoscaling.html>`_ or `manually <using-ray-on-a-cluster.html>`_.
```

Reply:
This was a bit confusing to me because you say "simply make sure that the Ray cluster is started" then end the sentence without indicating that you have to change the code to connect to it. I thought that was going to magically work - readers might stop there and think that too

Reply:
fixed.